config/kibana.yml

```diff
@@ -181,3 +181,6 @@
 # Maximum number of documents loaded by each shard to generate autocomplete suggestions.
 # This value must be a whole number greater than zero. Defaults to 100_000
 #unifiedSearch.autocomplete.valueSuggestions.terminateAfter: 100000
+
+
+uiSettings.experimental.themeSwitcherEnabled: true
```

Comment:
AFAIK, we shouldn't add config to the `kibana.yml`. Defaults must be set via the `defaultValue` in the config validation.

Reply:
Yes of course I will remove this. It is for allowing the designer to test the PR when deployed. Without that they can't turn on the settings...

Reply:
oopssss